.github/workflows/weekly-pulumi-update.yml

```diff
@@ -82,14 +82,6 @@ jobs:
 
         go mod tidy
 
-        cd ../sdk
-
-        go get github.com/pulumi/pulumi/sdk/v3
-
-        go mod download
-
-        go mod tidy
```

Comment:
I think you would have needed a -compat flag here since the CI job's go version is 1.18. We can revisit this when we re-add support for this separately though.